sdk/README.md

```diff
@@ -2,16 +2,39 @@
 
 ## Índice
 
-- [Introducción](#introducción)
-- [Consideraciones previas](#consideraciones-previas)
-- [Instalación y configuración](#instalación-y-configuración)
-  - [Instalación](#instalación)
-  - [Instalación de react-native-ssl-pinning](#instalación-de-react-native-ssl-pinning)
-  - [Configuración de redirect URI](#configuración-de-redirect-uri)
-- [Utilización](#utilización)
-- [Funcionalidades](#funcionalidades)
-- [Errores](#errores)
-- [Certificado *self-signed* en modo *testing*](#certificado-self-signed-en-modo-testing)
+- [SDK ID Uruguay React Native](#sdk-id-uruguay-react-native)
+  - [Índice](#índice)
+  - [Introducción](#introducción)
+  - [Consideraciones previas](#consideraciones-previas)
+  - [Instalación y configuración](#instalación-y-configuración)
+    - [Instalación](#instalación)
+    - [Instalación de react-native-ssl-pinning](#instalación-de-react-native-ssl-pinning)
+    - [Configuración de redirect URI](#configuración-de-redirect-uri)
+      - [Android](#android)
+      - [iOS](#ios)
+    - [Certificado *self-signed* en modo *testing*](#certificado-self-signed-en-modo-testing)
+  - [Utilización](#utilización)
+  - [Funcionalidades](#funcionalidades)
+    - [Función getParameters](#función-getparameters)
+    - [Función setParameters](#función-setparameters)
+      - [Errores setParameters](#errores-setparameters)
+    - [Función clearParameters y resetParameters](#función-clearparameters-y-resetparameters)
+    - [Función initialize](#función-initialize)
+      - [Errores initialize](#errores-initialize)
+    - [Función login](#función-login)
+      - [Errores login](#errores-login)
+    - [Función getToken](#función-gettoken)
+      - [Errores getToken](#errores-gettoken)
+    - [Función refreshToken](#función-refreshtoken)
+      - [Errores refreshToken](#errores-refreshtoken)
+    - [Función getUserInfo](#función-getuserinfo)
+      - [Errores getUserInfo](#errores-getuserinfo)
+    - [Función validateToken](#función-validatetoken)
+      - [Errores validateToken](#errores-validatetoken)
+    - [Función logout](#función-logout)
+      - [Errores logout](#errores-logout)
```

Comment:
Me parece un poco mucho el poner las secciones de errores en índice, no sé que opina el resto, pero to dejaría solo los ítems de cada función

Reply:
esto realmente no se como sucedió. Onda yo no lo hice jajaa

Reply:
Corregido